pkg/controllers/routing/network_routes_controller.go

```diff
@@ -341,6 +358,51 @@ func (nrc *NetworkRoutingController) updateCNIConfig() {
 			glog.Fatalf("Failed to insert `subnet`(pod CIDR) into CNI conf file: %s", err.Error())
 		}
 	}
+
+	if nrc.autoMTU {
+		err = nrc.autoConfigureMTU()
+		if err != nil {
+			glog.Fatalf("Failed to auto-configure MTU: %s", err.Error())
```

Comment:
Are we sure that we want to make this fatal here? We're defaulting MTU to enabled by default, and this will essentially terminate the `network_routes_controller` but above, when you try to set the MTU on the `kubeBridgeIf` you handle it with just an error in the logs.

We haven't been setting MTU so far. I think it would be better to just `glog.Errorf` here and warn people that we won't be able to set the MTU like we do in the `Run()` method above. That would essentially allow user's that encounter an error with this new default option to continue running kube-router the same way they have been running it and reduce the impact of this option quite a bit.

Reply:
Make sense to just keep it Errorf.